### PR TITLE
Improve the Readme

### PR DESCRIPTION
**Pull request by rposborne**
- [ ] Pass tests

We had no explanation now we do.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+## Vetcorp
+
+We manage fluffy vet practices.
```
Comment (rposborne) [inline README.md]: Why?

File: README.md [hunk 2]
```diff
@@ -1,3 +1,3 @@
 ## Vetcorp
 
-We manage fluffy vet practices.
+We manage fluffy vet practices, so they can improve their efficiency.
```
Comment (rposborne) [inline README.md:3]: This is wonderful

**Comment by MartinDally** [comment]
Awesome readme!!!

**Comment by clai88** [comment]
beautiful code.
very eloquent.

**Pull request merged**